### PR TITLE
Fix e2e test to verify that the filter is applied to twice summarized question

### LINKED ISSUE #15620: Filtering of twice summarized questions does not work in the simple question mode
When a question has nested aggregation in the custom question mode, it is not possible to apply a filter for this question in the simple question mode.

**To Reproduce**
1) Recreate custom question using Products table of the sample dataset and press "Visualize"
2) Try to filter by Category dimension

Screen Shot 2021-04-15 at 00 20 56

**Actual behavior**
It is not possible to save the filter with an entered value

**Expected behavior**
Make it possible to add a filter

**Information about your Metabase Installation:**
Latest `release-x.39.x` branch

### PR DESCRIPTION
**Pull request by uladzimirdev**
closes https://github.com/metabase/metabase/issues/15620

Fix repro for #15620 and unskip this test

### CODE REVIEW

**Comment by uladzimirdev** [comment]
@iethree I added you only because I found this thread https://github.com/metabase/metabase/pull/24164#discussion_r926045288, 😄

File: e2e/test/scenarios/filters/filter.cy.spec.js [hunk 1]
```diff
@@ -740,38 +740,48 @@ describe("scenarios > question > filter", () => {
...
-  it.skip("should work on twice summarized questions and preserve both summaries (metabase#15620)", () => {
+  it("should work on twice summarized questions and preserve both summaries (metabase#15620)", () => {
     visitQuestionAdhoc({
       dataset_query: {
         database: SAMPLE_DB_ID,
+        type: "query",
         query: {
           "source-query": {
-            "source-table": 1,
+            "source-table": PRODUCTS_ID,
```
Comment (nemanjaglumac) [inline e2e/test/scenarios/filters/filter.cy.spec.js:750]: 🙏

**Review by nemanjaglumac — APPROVED** [review]
Works like a charm!
Maybe just run it through the stress-test before the merge?

**Comment by uladzimirdev** [comment]
https://github.com/metabase/metabase/actions/runs/7653384491/job/20855134291

stress test is passed

**Pull request merged**